homeassistant/components/velbus/cover.py

```diff
@@ -26,10 +26,7 @@ async def async_setup_entry(
     """Set up Velbus switch based on config_entry."""
     await hass.data[DOMAIN][entry.entry_id]["tsk"]
     cntrl = hass.data[DOMAIN][entry.entry_id]["cntrl"]
-    entities = []
-    for channel in cntrl.get_all("cover"):
-        entities.append(VelbusCover(channel))
-    async_add_entities(entities)
+    async_add_entities([VelbusCover(channel) for channel in cntrl.get_all("cover")])
```

Comment:
```suggestion
    async_add_entities(VelbusCover(channel) for channel in cntrl.get_all("cover"))
```

Reply:
We recently found out that the generator expression isn't much faster than the list comprehension. See https://github.com/home-assistant/core/pull/112422

Reply:
It isn't a problem; it just makes no sense to do a list comprehension in this case. This PR adds it, and it is unneeded.

Please remove it.